app/src/main/java/com/example/roome/user_classes/RoommateAdditionalInfo.java

```diff
@@ -0,0 +1,80 @@
+package com.example.roome.user_classes;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.DialogFragment;
+
+import android.app.Dialog;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.view.Window;
+import android.view.WindowManager;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.example.roome.FirebaseMediate;
+import com.example.roome.R;
+import com.example.roome.UsersImageConnector;
+
+public class RoommateAdditionalInfo extends DialogFragment {
+    private ImageView exitOption, personImg;
+    private TextView additionalInfo, name, age;
+
+    ApartmentSearcherUser apartmentSearcherUser;
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+    }
+
+    @Nullable
+    @Override
+    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
+        Bundle bundle = getArguments();
+        apartmentSearcherUser = FirebaseMediate.getApartmentSearcherUserByUid(bundle.getString("apartmentId")); // todo
+        return inflater.inflate(R.layout.fragment_roommate_additional_info, container, false);
+    }
+
+    @Override
+    public void onActivityCreated(@Nullable Bundle savedInstanceState) {
+        exitOption = getView().findViewById(R.id.iv_btn_x);
+        exitOption.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                getDialog().dismiss();
+            }
+        });
+        additionalInfo = getView().findViewById(R.id.tv_additional_person_info);
```

Comment:
If you want you can extract these line to a method with an informative name so that it will be more clear what you are doing here..